ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -177,7 +181,23 @@ def nan_to_num(
     neginf: Optional[Union[float, int]] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    if copy:
+        t = x.clone()
+    if out is not None:
```

Comment:
since paddle doesn't natively support the out argument, I suggest we ignore it completely

Reply:
`local variable 't' referenced before assignment` because you didn't handle the case where `copy=False`
I'm suggesting again to use the tensorflow backend implementation which is much easier than this one and uses native functions

Reply:
sure 
I will implement nan_to_num in tensorflow backend implementation  and will add it via another pull request.
I have to see how to do it.
Thanks @MahmoudAshraf97 for being patient with me.